go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/cloudtrust/keycloak-client/v2
 go 1.24.1
 
 require (
-	github.com/cloudtrust/common-service/v2 v2.10.0
+	github.com/cloudtrust/common-service/v2 v2.10.1-0.20250716105739-e6ecdbb2996c
```

Comment:
Do not forget to change that before merging

Reply:
Done